crates/proof-of-sql/src/base/database/column.rs

```diff
@@ -234,28 +234,36 @@ pub enum ColumnType {
     /// Mapped to Curve25519Scalar
     #[serde(alias = "SCALAR", alias = "scalar")]
     Scalar,
+
+    /// Nullable version of the column type
+    #[serde(alias = "NULLABLE", alias = "nullable")]
+    Nullable(Box<ColumnType>),
```

Comment:
I think `ColumnType` and nullability should be orthogonal.

Reply:
by orthogonal do you mean that we should create a wrapper struct or enum?

Reply:
@ssddOnTop `ColumnType` shouldn't change. Instead `NullableColumn` should have a simple `is_nullable` method.

Reply:
@iajoiner what do you think should be the scope of this PR?

Shall I add/modify functions to perform column expr for this PR?